server/specs/server.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require('request');
+
+const serverUrl = 'http://127.0.0.1:7777';
+
+
+describe('Server', () => {
+
+
+  test('Should be able to handle GET request', (done) => {
+
+    request({
+      url: `${serverUrl}/booking`,
+      method: 'GET',
+    }, (error, response, body) => {
+      expect(response.statusCode).toBe(200);
+    });
+
+    done();
+  });
+
+  test('Should be able to handle POST request', (done) => {
+
+    request({
+      url: `${serverUrl}/booking`,
+      method: 'POST',
+    }, (error, response, body) => {
+      expect(response.statusCode).toBe(200);
+    });
+
+    done();
+  });
+
+  test('Should 404 when asked for a nonexistent file', (done) => {
```

Comment:
maybe this should test if it asks to GET for a booking of a nonexistent place instead? That way, you can test if your GET function handles for when someone tries to "book" a nonexistent listing instead of just checking if the express server works.

Reply:
@jennqiao Sure I'll add it to Trello ticket to handle